index.html

```diff
@@ -64,5 +64,214 @@ <h1 class="header__connect">
         </li>
       </ul>
     </section>
+
+    <!-- /**
+        *! Works Section Here.
+        **/ -->
+
+    <section class="works">
+      <div class="works__cards">
+        <img src="./images/card-1.svg" alt="" />
+        <h1 class="project__title">Tonic</h1>
```

Comment:
Dear @Zuhaib042, thank you very much for your feedback and suggestion. however, your requested change is not part of this project.  It will only be applicable when we add media queries and cards for the "desktop version"

Reply:
@tsheporamantso I am really sorry about that. that was actually a misunderstanding on my part. I really apologize for my shortcoming.